### ISSUE TITLE
Error: google-github-actions/auth failed with: retry function failed after 1 attempt: failed to parse service account key JSON credentials: unexpected token \ in JSON at position 1

### ISSUE THREAD

**Issue by rahul4tem** [google-github-actions/auth#316]
### TL;DR

I am fetching raw JSON file similar to (https://github.com/google-github-actions/auth/issues/315) from vault secrets in github actions, but the format is getting changed while trying to retrieve with this action. The issue started yesterday, I had tried with github secrets and is working fine, but not with vault secrets. 
No change is done in JSON file/configurations.

### Expected behavior

Proper JSON format for vault secrets for github actions

### Observed behavior

Original JSON file from vault is getting changed to {\n"type": "**********",\n"project_id": "***",\n"private_key_id":........\n} while fetching in github actions

### Action YAML

```yaml
- id: auth
      uses: google-github-actions/auth@v0
      with:
        credentials_json: ${{ steps.import-secrets.outputs.GOOGLE_CREDS }}
 ###Tried with env.GOOGLE_CREDS as well
```

### Log output

```text
Error: google-github-actions/auth failed with: retry function failed after 1 attempt: failed to parse service account key JSON credentials: unexpected token \ in JSON at position 1
```

### Additional information

The configuration was working fine till yesterday and nothing has been changed from our end. I was monitoring the ticket (**https://github.com/google-github-actions/auth/issues/315**), but I want to fetch from vault and not from Github secrets. 
**Note**: with Github secrets this is working fine

Existing file format: 

{
  "type": "service_account",
  "project_id": "PROJECT_ID",
  "private_key_id": "KEY_ID",
  "private_key": "-----BEGIN PRIVATE KEY-----\nPRIVATE_KEY\n-----END PRIVATE KEY-----\n",
  "client_email": "SERVICE_ACCOUNT_EMAIL",
  "client_id": "CLIENT_ID",
  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
  "token_uri": "https://accounts.google.com/o/oauth2/token",
  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/SERVICE_ACCOUNT_EMAIL"
}

**Comment by sethvargo** [comment]
Hi @rahul4tem - this feels like a duplicate of #315. What is the value of `steps.import-secrets.outputs.GOOGLE_CREDS`? #315 was also about fetching from Vault, so I wonder if there was a Vault API change.

**Comment by sethvargo** [comment]
Duplicate of https://github.com/hashicorp/vault-action/issues/469

**Comment by sdsani** [comment]
I am able to test/validate and this issue is result of #469 mentioned above. Using 2.6.0 solves this problem so will use that.
